src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/AwsCliIT.kt

```diff
@@ -0,0 +1,75 @@
+package com.atlassian.performance.tools.awsinfrastructure
+
+import com.amazonaws.services.ec2.model.InstanceType
+import com.atlassian.performance.tools.aws.api.Investment
+import com.atlassian.performance.tools.aws.api.SshInstance
+import com.atlassian.performance.tools.aws.api.SshKeyFormula
+import org.assertj.core.api.Assertions
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import java.time.Duration
+import java.util.*
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit
+
+class AwsCliIT {
+
+    private val workspace = IntegrationTestRuntime.taskWorkspace.isolateTest(javaClass.simpleName)
+    private val aws = IntegrationTestRuntime.aws
+    private lateinit var executor: ExecutorService
+    private lateinit var sshInstance: SshInstance
+
+    @Before
+    fun setUp() {
+        executor = Executors.newCachedThreadPool()
+        val nonce = UUID.randomUUID().toString()
+        val lifespan = Duration.ofMinutes(30)
+        val keyFormula = SshKeyFormula(
+            ec2 = IntegrationTestRuntime.aws.ec2,
+            workingDirectory = workspace.directory,
+            lifespan = lifespan,
+            prefix = nonce
+        )
+        val investment = Investment(
+            useCase = "AwsCli Integration test",
+            lifespan = lifespan
+        )
+        val networkFormula = NetworkFormula(investment, aws)
+        val network = networkFormula.provision()
```

Comment:
Should I clean the network after the test run? How can I do that? 
(+ the same question about the sshKey).

Reply:
Network: https://ecosystem.atlassian.net/browse/JPERF-517
Hmm, it seems we always relied on housekeeping to clean up the keys. I guess they don't cost anything, but there is a limit.